packages/calcite-components/src/components/block/block.scss

```diff
@@ -29,13 +30,21 @@
 }
 
 .header-container {
-  @apply grid items-stretch;
+  @apply grid items-stretch w-full;
   grid-template: auto / auto 1fr auto auto;
   grid-template-areas: "handle header control menu";
   grid-column: header-start / menu-end;
   grid-row: 1 / 2;
 }
 
+.header ::slotted(calcite-icon) {
+  margin-inline-end: var(--calcite-spacing-md);
+}
+
+.header .icon ::slotted([slot="icon"]) {
```

Comment:
I don't think these styles are reachable as CSS.icon isn't used anywhere.

Reply:
`CSS.icon` is used to wrap `loader, status-icon, and icon-slot`, we keep them as deprecated for now